data/coalition/coalition missions.txt

```diff
@@ -1393,3 +1393,63 @@ mission "Arachi Cartoonist 2"
 				accept
 			label noship
 			`	"Ah, I see. Well, just a reference it would be, so the name matters not, I suppose. Anyway, thanks again Captain."`
+
+
+
+mission "Coalition: Alpha Encounter"
+	minor
+	invisible
+	to offer
+		has "Coalition: First Contact: done"
+		has "remembrance day: offered"
+		random < 1
+	source
+		government "Coalition"
+		not attributes "station"
+		not planet "Secret Sky" "Remote Blue" "Bloptab's Furnace" "Ahr"
+	on offer
+		log "Minor People" "Alitis" `Alitis, formerly known as Athis, is an Alpha who lives in the Coalition. Centuries ago, he stole a jump drive from an alien ship and used it to explore, but after some sort of encounter in the galactic east, he decided to give up on his former life.`
+		conversation
+			`You walk around the spaceport for a few minutes, when suddenly you're tackled into the ground, with whoever attacked you pinning you down, holding your arms behind your back to restrain you. When you manage to look back, you see a man of human appearance, with very long hair and wearing an intricate necklace of gold. His most notable trait, however, is his green skin - an Alpha.`
+			`	Your eyes don't meet for long, though, as he looks to his side and lets go of your arms to try and block an Arach that swoops in with a tackle, knocking him off of you. They wrestle for a while, as you get up, but it doesn't take long for the Arach Heliarch to restrain the Alpha, with all eight legs firmly holding his arms to his torso, as the Arach rests her entire body weight on the Alpha's back. A Saryd Heliarch approaches you, telling you that "Keep your distance, you must," and then looking at the Alpha. "What reason for attacking this person, did you have?"`
```

Comment:
This Arach would be easily toppled, any more than 6 legs is asking to end up on the floor.

Reply:
Eh, they're heavy and wide enough I doubt it'd be an easy task.

Reply:
If they're getting all 8 legs onto a single person's torso, then they've created a fulcrum. It doesn't matter how heavy they are, in fact the heavier they are the worse off they are. It would be trivial to topple them. They require at least two other points of contact with something that is not their target in order to safely and effectively restrain them.

Reply:
They've got long legs, a lot of joints would be touching the ground.

Reply:
They can't apply their weight through those legs, if they're touching the ground some other way with a different joint. That's not how this works.